changelog/11081.improvement.rst

```diff
@@ -1 +1,7 @@
 The :confval:`norecursedir` check is now performed in a :hook:`pytest_ignore_collect` implementation, so plugins can affect it.
+
+If after updating to this version you see that your `norecursedir` setting is not being respected,
+it means that a conftest or a plugin you use has a bad `pytest_ignore_collect` implementation.
+Most likely, your hook returns `False` for paths it does not want to ignore,
+which ends the processing and doesn't allow other plugins, including pytest itself, to ignore the path.
+The fix is simple - return `None` instead of `False` for paths your hook doesn't want to ignore.
```

Comment:
```suggestion
The fix is to return `None` instead of `False` for paths your hook doesn't want to ignore.
```
Feel friendlier if you don't think this is a "simple" because you're a beginner.

Reply:
Agreed, thanks!